lib/draw.c

```
@@ -23,7 +26,7 @@ mesh importMeshFromOBJFile (char * pathToFile)
     if (NULL == obj) 
     {
         printf("Error: .OBJ file not found\n");
-        printf("Try to run the program from top level \"Cube/\" dir\n");
+        printf("Reminder: run the program from top level \"Cube/\" dir\n");
         return newMesh;
```

I would probably instead throw an error here if file not found